src/js/view/view.js

```diff
@@ -337,7 +337,7 @@ function View(_api, _model) {
         playerViewModel.change('controls', changeControls);
         _model.change('streamType', _setLiveMode);
         _model.change('mediaType', _onMediaTypeChange);
-        playerViewModel.change('playlistItem', onPlaylistItem);
+        playerViewModel.change('playlistItem', onPlaylistItem.bind(this));
```

Comment:
I prefer arrow functions over bind so we can see what's passed, and not be open to `Function.prototype.bind` being tampered with.

`playerViewModel.change('playlistItem', (model, item) => { onPlaylistItem(model, item); });`

Reply:
done